SimpleMania/css/style.css

```diff
@@ -0,0 +1,223 @@
+@font-face {
+    font-family: "Sica 3";
+    src: url("../fonts/SicaExtraBold.otf");
+}
+body,html{
+    padding: 0;
+    margin: 0;
+    overflow-y: hidden;
+}
+#main{
+    font-size: 20px;
+    font-family: "Sica 3", Helvetica, sans-serif;
+    font-weight: normal;
+    color: white;
+    width: 460px;
+    height: 550px;
+    padding: 20px;
+    position: relative;
```

Comment:
highly recommend to remove those and right,bottom in #box_right cause it's useless in that case